microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc

```diff
@@ -6,25 +6,25 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-You can automatically restore data from manual backups when {microshift-short} fails to start by using the `auto-recovery` feature.
+You can automatically restore data from manual backups when {microshift-short} fails to start by using the `auto-recovery` feature. Use the following options with the existing `backup` and `restore` commands:
 
-You can use the following options with the existing `backup` and `restore` commands in this feature:
+* The `--auto-recovery` option selects the most recent version of the backup, and then restores it. This option treats the `PATH` argument as a path to a directory that holds all the backups for automated recovery, and not just as a path to a particular backup file.
 
-* `--auto-recovery`: Selects the most recent version of the backup, and then restores it. This option treats the `PATH` argument as a path to a directory that holds all the backups for automated recovery, and not just as a path to a particular backup file.
-* `--dont-save-failed`: Disables the backup of failed {microshift-short} data.
+* The `--dont-save-failed` option disables the backup of failed {microshift-short} data.
```

Comment:
I think we should include that this is for `microshift restore --auto-recovery` only. I feel like some context is missing: `restore` will make a copy/backup of microshift data (/var/lib/microshift) that was failing to start.

I know we have that in the NOTE below, but feels like reading it in wrong order

Reply:
I created a concept module for this so we can explain a bit more.